src/server/MCFadapter.js

```diff
@@ -5,41 +5,67 @@ export class MCFAdapter {
   pool = null
   binsHash = {}
 
-  setPool = dbObj => {
+  setPool = (dbObj, mongodbObj) => {
     this.pool = dbObj
+    this.mongodb = mongodbObj
     return 0
   }
 
   // Computes smooth histogram curve
   bellCurve = (sample, subsets, type) => {
     if (type === 'fpkm') {
-      return this.pool
-        .query(`SELECT ${sample}_log2 FROM mcf10a_vs_mcf7 WHERE ${sample}_log2 != 'Infinity'`)
+      const key = sample + '_log2'
+      const conditions = {}
+      const select = {}
+      conditions[key] = { $ne: Infinity }
```

Comment:
With regards to using variable names as keys for objects, see this SO question: https://stackoverflow.com/questions/11508463/javascript-set-object-key-by-variable

We use ES6, so we can do this:

```
conditions = {
    [key]: {
        $ne: Infinity
     }
}
```

which looks more readable.